test/generate-plugin.test.js

```diff
@@ -100,16 +100,35 @@ function define (t) {
   })
 
   test('should finish succesfully', async (t) => {
-    t.plan(17 + Object.keys(expected).length)
+    t.plan(18 + Object.keys(expected).length)
     try {
       await generate(workdir, pluginTemplate)
       await verifyPkg(t)
       await verifyCopy(t, expected)
+      await verifyFastifyPluginVersion(t)
     } catch (err) {
       t.error(err)
     }
   })
 
+  function verifyFastifyPluginVersion (t) {
+    return new Promise((resolve, reject) => {
+      const pkgFile = path.join(workdir, 'package.json')
```

Comment:
we can simplify it:
```suggestion
      const pkg = require(path.join(workdir, 'package.json'))
```

Reply:
I wrote the test like this to match other test styles in the file. But if it is okay for you, @Eomm, I could also change the behavior of other test functions